<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTableDB</class>
 <widget class="QDialog" name="DialogTableDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Товары</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="5">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonEdit">
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>121</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonDelete">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>DialogTableDB</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
